
Training STarted!!!!!!!!!!
Loaded blender (138, 400, 400, 4) torch.Size([40, 4, 4]) [400, 400, 555.5555155968841] ./data/nerf_synthetic/lego
Not ndc!
Begin
TRAIN views are [ 0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20 21 22 23
 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47
 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71
 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95
 96 97 98 99]
TEST views are [113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130
 131 132 133 134 135 136 137]
VAL views are [100 101 102 103 104 105 106 107 108 109 110 111 112]
[Config] Center cropping of size 200 x 200 is enabled until iter 500
blender_paper_lego 5 [3.9039872] 0.52773833 4
iter time 0.14746
test poses shape torch.Size([25, 4, 4])
/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/torch/functional.py:507: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3549.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
0 0.000102996826171875
torch.Size([400, 400, 3]) torch.Size([400, 400])
1 10.47217845916748
2 10.486451864242554
3 10.508649349212646

4 10.530432224273682
5 10.569321155548096
6 10.588382005691528
7 10.60591197013855
8 10.62427544593811
9 10.647185325622559
10 10.664222002029419
11 10.673948287963867
12 10.659646272659302
13 10.658951759338379
14 10.669053077697754
15 10.690602540969849
16 10.70813536643982
17 10.766984462738037
18 10.784195184707642
19 10.781676054000854
20 10.782905101776123
21 10.785690784454346
22 10.788499116897583
23 10.786435604095459
24 10.778846263885498
Saved test set
blender_paper_lego 10 [4.546131] 0.5042807 9
iter time 0.14753
Traceback (most recent call last):
  File "/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 61, in _wrapfunc
    return bound(*args, **kwds)
TypeError: clip() received an invalid combination of arguments - got (int, int, out=NoneType), but expected one of:
 * (Tensor min, Tensor max)
 * (Number min, Number max)
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "profile_nerf.py", line 552, in run
    train(args)
  File "profile_nerf.py", line 896, in train
    image = to8b(image)#.permute(2,0,1))
  File "/scratch/vb2386/Documents/nerf-pytorch/run_nerf_helpers.py", line 11, in <lambda>
    to8b = lambda x : (255*np.clip(x,0,1)).astype(np.uint8)#lambda x: (255 * np.clip(x.cpu().detach().numpy(), 0, 1)).astype(np.uint8)
  File "<__array_function__ internals>", line 5, in clip
  File "/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2084, in clip
    return _wrapfunc(a, 'clip', a_min, a_max, out=out, **kwargs)
  File "/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 70, in _wrapfunc
    return _wrapit(obj, method, *args, **kwds)
  File "/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 47, in _wrapit
    result = getattr(asarray(obj), method)(*args, **kwds)
  File "/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/numpy/core/_asarray.py", line 85, in asarray
    return array(a, dtype, copy=False, order=order)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/torch/_tensor.py", line 1062, in __array__
    return self.numpy()
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.